lightning-c-bindings/src/c_types/mod.rs

```diff
@@ -0,0 +1,78 @@
+use bitcoin::Script as BitcoinScript;
+use bitcoin::secp256k1::key::PublicKey as SecpPublicKey;
+use bitcoin::secp256k1::key::SecretKey as SecpSecretKey;
```

Comment:
I think in theory, yea, they wrap underlying ffi/C types explicitly and you can use those, though that then assumes the caller is in C and/or using libsecp under the hood. I presume that this would be annoying to deal with if you were not, and byte arrays are universal. That said, it *does* result in an extra copy or two, including for private keys, which kinda sucks.